pkg/compose/build.go

```diff
@@ -163,7 +167,19 @@ func (s *composeService) getBuildOptions(project *types.Project, images map[stri
 
 }
 
-func (s *composeService) getLocalImagesDigests(ctx context.Context, project *types.Project) (map[string]string, error) {
+func (s *composeService) getLocalImagesIDs(ctx context.Context, project *types.Project) (map[string]string, error) {
```

Comment:
Image ID is different from `RepoDigests` values.. I was confused and renamed it.